app/backend/src/couchers/servicers/references.py

```diff
@@ -221,7 +221,7 @@ def WriteHostRequestReference(self, request, context):
             session.commit()
 
             # send the recipient of the reference an email
-            send_host_reference_email(reference, both_written=other_reference is not None)
+            send_host_reference_email(reference, context, host_request, reference.reference_type, both_written=other_reference is not None)
```

Comment:
The arguments are wrong given the method signature

```py
 def send_host_reference_email(reference, other_user, host_request, surfed, both_written):
```

Reply:
This is fixed I think:
other_user is being passed in as reference.from_user.
surfed is changed to false, because this is just for the host I believe.